Source/Core/QuantizedMeshTerrainData.js

```diff
@@ -483,7 +483,7 @@ define([
             return interpolateHeight(this, u, v);
         }
 
-        interpolateMeshHeight(this, u, v);
+        return interpolateMeshHeight(this, u, v);
```

Comment:
Does this have anything to do with #3411 or any of the other height query issues we've seen?